### PR TITLE
Contributor

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
Original PR: https://github.com/spryker/spryker-docs/pull/1160
TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.md [hunk 1]
```diff
@@ -4,28 +4,32 @@ description: Learn about the development, staging, and production environments o
...
 
 ### Production (PROD)
 
-Your production environment should host the code that the end-user gets to interact with directly. The code should already be tested on the staging environment for stability and performance. Please do not make changes to the production environments directly unless they have been verified on your lower environments first.
+Your production environment must host the code that the end-user gets to interact with directly. The code must already be tested on the staging environment for stability and performance. 
+
+{% info_block infoBox "Note" %}
+
+Do not make changes to the production environments directly unless they have been verified on your lower environments first.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.md:30]: local?

Comment (VadymSachenko) [reply docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.md:30]: @michalwroblewski-spryker

Comment (VadymSachenko) [reply docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.md:30]: @andriitserkovnyi Michal Wroblewski says "it is lower (Staging / Dev)"

Comment (VadymSachenko) [inline docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.md:30]:
```suggestion
Do not make changes to the production environments directly unless they have been verified on your lower [environments](/docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.html) (staging or dev) first.
```

File: docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.md [hunk 2]
```diff
@@ -4,28 +4,32 @@ description: Learn about the development, staging, and production environments o
...
 
 ### Production (PROD)
 
-Your production environment should host the code that the end-user gets to interact with directly. The code should already be tested on the staging environment for stability and performance. Please do not make changes to the production environments directly unless they have been verified on your lower environments first.
+Your production environment must host the code that the end-user gets to interact with directly. The code must already be tested on the staging environment for stability and performance. 
+
+{% info_block infoBox "Note" %}
+
+Do not make changes to the production environments directly unless they have been verified on your lower [environments](/docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.html) (staging or dev) first.
```
Comment (andriitserkovnyi) [inline docs/cloud/dev/spryker-cloud-commerce-os/environments-overview.md:30]: Why provide a link to the doc you are already reading? Just replace `lower environments` with `staging and dev environments`.

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**